### PR TITLE
added option for hiding separators

### PR DESCRIPTION
**Pull request by robotbrad**
and adjusted default settings

### CODE REVIEW

File: css/pane_tab_content.css [hunk 1]
```diff
@@ -25,7 +25,7 @@ body.styled-scrollbars .oblogger-content {
     padding: 0 4px 70px;
 }
 
-.oblogger-content .hidden { display:none; }
+.oblogger-content .hidden { display:none !important; }
```
Comment (msphair) [inline css/pane_tab_content.css:28]: Was this the linch pin that was making it not work yesterday?

File: src/settings.ts [hunk 2]
```diff
@@ -89,7 +91,9 @@ const UPGRADE_FUNCTIONS: {[id: number]: (settings: ObloggerSettings) => void } =
         const newSettings = settings as ObloggerSettings_v2;
         if (newSettings) {
             newSettings.vaultVisible = true;
-            newSettings.clockVisible = true;
+            newSettings.clockVisible = false;
```
Comment (msphair) [inline src/settings.ts:94]: Why have the clock off by default?

**Review by msphair — APPROVED** [review]

File: css/pane_tab_content.css (hunk 1 above)
Comment (robotbrad) [reply css/pane_tab_content.css:28]: it was! i could keep moving around the .hidden css whenever we use it (or write multiple of them) but labeling it important should just make it work regardless of its location.

File: src/settings.ts (hunk 2 above)
Comment (robotbrad) [reply src/settings.ts:94]: it just seems appropriate that the default behavior is to have it off and that people should turn it on if they want it on. it seems like an "extra" feature.

**Pull request merged**